tests/data/test456

```diff
@@ -0,0 +1,33 @@
+<testcase>
+<info>
+<keywords>
+HTTP
+variables
+</keywords>
+</info>
+
+#
+# Client-side
+<client>
+<server>
+http
+</server>
+<name>
+Variable output starting with null byte
+</name>
+<file name="%LOGDIR/junk">
```

Comment:
I have used `%LOGDIR/junk` here because that is the file all the `test45?` tests are using, but I am now wondering if this is fine.
Wouldn't this possibly cause problems when running tests in parallel?

Reply:
When run in parallel, each version gets its own %LOGDIR so I think it is fine.